### ISSUE TITLE
Flye stuck at scaffolding step for large plant genome

### ISSUE THREAD

**Issue by danielle-khost** [fenderglass/ABruijn#353]
Hi there!
I am attempting to use flye (version 2.8.1-b1676) to assemble a fairly tricky plant genome (~6.6 Gbase size, repeat dense). I set it running a while back (12/7/20) on our local machine that has ~1 TB of RAM using 30 threads and default parameters. It has run pretty steadily since then and had gotten "stuck" before at certain steps (especially the disjointig assembly step, which took almost a month), so I decided to be patient and let it run. However, it now seems to be stuck again at the final scaffolding step, which has also been running almost a month now, specifically the "Writing GFA" step. Here is the tail of the log file:

```
[2021-01-17 03:32:36] DEBUG: Final graph contiain 115126 egdes
[2021-01-17 03:32:36] DEBUG: Extending contigs into repeats
[2021-01-17 03:37:28] DEBUG: Covered 22274 repetitive contigs
[2021-01-17 03:37:28] INFO: Generated 92857 contigs
[2021-01-17 03:37:31] DEBUG: Writing FASTA
[2021-01-17 04:02:15] DEBUG: Generating scaffold connections
[2021-01-17 04:02:17] INFO: Added 123 scaffold connections
[2021-01-17 04:02:17] DEBUG: Writing Dot
[2021-01-17 20:22:00] DEBUG: Writing FASTA
[2021-01-17 20:25:49] DEBUG: Writing Gfa
```

As far as I can tell, it is still writing to the GFA file (which is now 190 GB), so I know it isn't completely stalled. My question is whether this is typical behavior, and whether I should try cancelling and resuming the job to un-stick it or if I should just be patient longer? Alternatively, could I cancel the job and scaffold the contigs that flye has produced myself?

Thank you so much!
-Danielle

**Comment by EinarBaldvin** [comment]
Hi Danielle,

I would highly recommend that you upgrade Flye to a newer version. I had this exact same problem with a large repetitive plant genome, but with 2.8.2 this issue was solved and gfa generation is by orders of magnitude faster.

Best,
Einar

**Comment by danielle-khost** [comment]
Oh awesome, thank you for the tip! That new version must have come out just as I had started the assembly running, unfortunate timing :/
Do you happen to know whether the versions of Flye are compatible? I.e. could I resume the assembly from where the previous version left off? Or would I have to start over from scratch?

**Comment by EinarBaldvin** [comment]
I think the fix came in 2.8.1-b1683, and I am fairly sure you can resume from the last checkpoint with the newer Flye version as it is within the 2.8.x release.

**Comment by mikolmogorov** [comment]
Yep, 2.8.2+ should have this issue resolved now. There some extra information here, including how to restart/recover the existing run: https://github.com/fenderglass/Flye/issues/290

Thanks,
Mikhail
